feature/platform/transceiver/zr_tunable_parameters_test/README.md

```diff
@@ -0,0 +1,137 @@
+      absolute accuracy should be within +/- 1 dBm of the target value.
+    * For reported data check for validity: min <= avg/instant <= max
+
+   
+## TRANSCEIVER-5.2
+
+*   When the modules or the devices are still in a boot stage, they must not
+    stream any invalid string values like "nil" or "-inf".
+
+*   Frequency must be specified as uint64 in MHz. Streamed values for frequency
+    offset must be of type decimal64.
+
+*   TX Output power must be of type decimal64.
+
+## TRANSCEIVER-5.3
+
+*   Verify that the optics Tunable Frequency and TX output power tunes back to
+    the correct value as per configuration after the interface flaps.
+
+    *   Enable a pair of ZR interfaces on the DUT as explained above.
+    *   Verify the ZR optics frequency and TX output power telemetry values are
+        set in the normal range.
+    *   Disable or shut down the interface on the DUT.
+    *   Verify with interfaces in down state both optics are streaming uint 0
+        value for frequency and decimal64 for TX output power.
+    *   Re-enable the interfaces on the DUT.
+    *   Verify the ZR optics tune back to the correct frequency and TX output
+        power as per the configuration and related telemetry values are updated
+        to the value in the normal range again.
+
+* With above test also verify
+    * Laser frequency offset should not be more than +/- 1.8 GHz max from the
+      configured centre frequency.
+    * When set to a specific target output power, transmit power control
+      absolute accuracy should be within +/- 1 dBm of the target configured
+      output power.
+    * For reported data check for validity: min <= avg/instant <= max
+
+## TRANSCEIVER-5.4
+
+*   Verify that the optics Tunable Frequency and TX output power tunes back to
+    the correct value as per configuration after a fiber cut.
+
+    *   Enable a pair of ZR interfaces on the DUT as explained above.
+    *   Verify the ZR optics Frequency and TX output power telemetry values are
+        in the normal range.
+    *   Simulate a fiber cut using the optical switch that sits in-between the
+        DUT ports.
+    *   Verify with link in down state due to fiber cut both optics are streaming
+        uint64 for frequency and decimal64 for TX output power. 
+    *   Re-enable the optical switch connection to clear the fiber cut fault.
+    *   Verify the ZR optics is able to stay tuned to the correct frequency and
+        TX output power as per the configuration.
+
+* With above test also verify
+    * Laser frequency offset should not be more than +/- 1.8 GHz max from the
+      configured centre frequency.
+    * When set to a specific target output power, transmit power control
+      absolute accuracy should be within +/- 1 dBm of the target configured
+      output power.
+    * For reported data check for validity: min <= avg/instant <= max
+
+**Note:** For min, max, and avg values, 10 second sampling is preferred. If 
```

Comment:
Do you also want to get [/components/component/transceiver/physical-channels/channel/state/output-power/instant](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-platform.html#components-component-transceiver-physical-channels-channel-state-output-power-instant) ?

Reply:
Unlike RX input power there is no concept of signal power vs total power for the TX output power applicable here